examples/users.js

```diff
@@ -21,6 +21,15 @@ zulip(config)
     //        email: 'john.doe@exmaple.com' },
     //   ...
     z.users.retrieve().then(console.log);
+
+    // Create a new user
+    const createParams = {
+      email: 'newbie@zulip.com',
+      password: 'temp',
+      full_name: 'New User',
+      short_name: 'newbie',
+    };
+    z.users.create(createParams).then(console.log);
```

Comment:
Nit: probably a line break after this line.

Reply:
Done! Thanks! :)